php-checks/src/test/resources/checks/regex/SingleCharCharacterClassCheck.php

```diff
@@ -7,10 +7,10 @@ class SingleCharCharacterClassCheck
   function noncompliant($input) {
     preg_match('/[0]/', $input);  // Noncompliant {{Replace this character class by the character itself.}}
     //            ^
-
-    // The following couple of testcases would have been Noncompliant *if* PHP regexes supported nested char classes:
-    // preg_match('/[[a]]/', $input);  // Noncompliant {{Replace this character class by the character itself.}}
-    // preg_match('/[1-2[3]4-5]/', $input);  // Noncompliant {{Replace this character class by the character itself.}}
+    preg_match('/[z]/', $input);  // Noncompliant {{Replace this character class by the character itself.}}
+    //            ^
+    preg_match('/[ ]/', $input);  // Noncompliant {{Replace this character class by the character itself.}}
```

Comment:
It's okay. I see, it's quite hard to be very creative for such test cases. Nevertheless, you don't have to repeat the message if it changed.